enterprise/coordinator/peerdiscovery/peerdiscovery.go

```diff
@@ -14,9 +14,22 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+type Discovery struct {
+	client    kubernetes.Interface
+	namespace string
+}
+
+func New(client kubernetes.Interface, namespace string) *Discovery {
```

Comment:
This is exported and should have comments, right? (like I said, expected golangci-lint to fail)

Reply:
Looks like that check is still disabled, then.